src/components/shared/NavTabs/NavTabs.tsx

```diff
@@ -0,0 +1,25 @@
+import classes from "./NavTabs.module.css";
+
+interface NavigationTabsProps<T extends string> {

```

Comment:
Кажется, можно так, табов ограниченное количество вариантов

```suggestion
interface NavigationTabsProps<T extends TabType> {

```